### ISSUE TITLE
Change Noise-Control Mode 

### ISSUE THREAD

**Issue by StreamingKarl** [d4rken-org/capod#25]
> AirPods Pro and AirPods Max have three noise-control modes: Active Noise Cancellation, Transparency mode, and Off. You can switch between them, depending on how much of your surroundings you want to hear.
 [https://support.apple.com/en-us/HT210643](https://support.apple.com/en-us/HT210643)

It would be nice if CAPod could update the mode.

**Comment by surfaceflinger** [comment]
Probably impossible until someone will figure out write operations // https://github.com/d4rken-org/capod/issues/22#issuecomment-1221848272

**Comment by d4rken** [comment]
Yes. Unfortunately Apple is not really generous with sharing Infos about this.

**Comment by d4rken** [comment]
Requires #215
